### ISSUE TITLE
[Update Request]: WhatPulse 4.0

### ISSUE THREAD

**Issue by f4bio** [microsoft/winget-pkgs#112653]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

WhatPulse.WhatPulse

### Package Version

4.0.0

### Please describe the changes you would like to see

https://whatpulse.org/downloads/

**Comment by Trenly** [comment]
I'm not able to find any silent install switches for the 4.0 version. Do you happen to know them?

**Comment by f4bio** [comment]
> I'm not able to find any silent install switches for the 4.0 version. Do you happen to know them?

Not really, sorry. Might that be the reason nobody tried to provide an updated installation?
Thats too bad... I guess this issue can be closed then?!
